src/schema/v2/city/index.ts

```diff
@@ -23,141 +20,159 @@ import {
 } from "./constants"
 import { convertConnectionArgsToGravityArgs } from "lib/helpers"
 import Near from "schema/v2/input_fields/near"
-import { LatLng, Point, distance } from "lib/geospatial"
+import { LatLng, distance } from "lib/geospatial"
 import { ResolverContext } from "types/graphql"
 import { allViaLoader, MAX_GRAPHQL_INT } from "lib/all"
 import { StaticPathLoader } from "lib/loaders/api/loader_interface"
 import { BodyAndHeaders } from "lib/loaders"
 import { sponsoredContentForCity } from "lib/sponsoredContent"
 
+export interface TCity {
+  slug: string
+  name: string
+  full_name: string
+  /** [lat, lng] */
+  coords: [number, number]
+}
+
 const PartnerShowPartnerType = new GraphQLEnumType({
   name: "PartnerShowPartnerType",
   values: {
-    GALLERY: {
-      value: ["Gallery"],
-    },
-    MUSEUM: {
-      value: ["Institution", "Institutional Seller"],
-    },
+    GALLERY: { value: ["Gallery"] },
+    MUSEUM: { value: ["Institution", "Institutional Seller"] },
   },
 })
 
-const CityType = new GraphQLObjectType<any, ResolverContext>({
+export const CityType = new GraphQLObjectType<TCity, ResolverContext>({
   name: "City",
-  fields: {
-    slug: {
-      type: GraphQLString,
-    },
-    name: {
-      type: GraphQLString,
-    },
-    coordinates: {
-      type: LatLngType,
-    },
-    showsConnection: {
-      type: ShowsConnection.connectionType,
-      args: pageable({
-        sort: {
-          type: ShowSorts,
-        },
-        status: {
-          type: EventStatus.type,
-          defaultValue: "CURRENT",
-          description: "Filter shows by chronological event status",
-        },
-        partnerType: {
-          type: PartnerShowPartnerType,
-          description: "Filter shows by partner type",
-        },
-        dayThreshold: {
-          type: GraphQLInt,
-          description:
-            "Only used when status is CLOSING_SOON or UPCOMING. Number of days used to filter upcoming and closing soon shows",
-        },
-        includeStubShows: {
-          type: GraphQLBoolean,
-          description: "Whether to include local discovery stubs",
+  fields: () => {
+    const { ShowsConnection } = require("../show")
+    const { fairConnection } = require("../fair")
```

Comment:
**non-blocking:** Curious why these imports changed to this style?

Reply:
Apparently dependencies get resolved in a different order when running tests (?) and there was some circular dependencies when running the specs, so had to thunk/defer the require in order to get anything to load.